docs/developer/cloning-databases.md

```diff
@@ -0,0 +1,5 @@
+# Cloning Databases
+The clone-db workflow clones a Source database to a Destination database using cloud.gov's cg-manage-rds tool. This document contains additional information needed to understand how the workflow functions.
+
+## Additional Roles Required
```

Comment:
@Matt-Spence can you add a section for "how to turn off the db copy in an emergency" with explicit steps for what people should do. If there were a time sensitive issue I think it may take a few mins to process what is being said here and the reverse engineer the commands to turn it off. It would help in a time sensitive situation to just have clear steps to follow for turning it off. I think it would be something like the following
Step 1:
Get  the name of the correct service
```cf spaces-users cisa-dotgov staging ```
output should look like:
<provide an example>
The space user you want should look like xyz-example

step 2:
Remove the space develeper role by doing the following command
...

Reply:
Can do, but one question: what value does that bring that pushing a change to the workflow itself doesn't?

Reply:
The change to the workflow would require a PR review which immediately slows it down in an emergency as it adds the extra step of putting a PR up, asking someone to approve and waiting for them to see your message( or call) as well as waiting for them to look it over and approve. While you make a good point, this doc should mention "make a PR that deactivates this workflow"as an alternative way to stop the workflow; I do think the list of cf steps would still be better to have in the event that time is really critical.